firefox-ios/firefox-ios-tests/Tests/XCUITests/URLValidationTests.swift

```diff
@@ -42,9 +42,7 @@ class URLValidationTests: BaseTestCase {
     }
 
     private func clearURL() {
-        if iPad() {
-            navigator.goto(URLBarOpen)
-            app.buttons["Clear text"].waitAndTap()
-        }
+        navigator.goto(URLBarOpen)
+        app.buttons["Clear text"].waitAndTap()
```

Comment:
On iPhones, the text on the URL bar is not cleared intermittently. Let's tap "x" just to be sure.